setuptools/command/easy_install.py

```diff
@@ -1717,7 +1717,7 @@ def add(self, dist):
             dist.location not in self.sitedirs
             or
             # account for '.' being in PYTHONPATH
-            dist.location == os.getcwd()
+            dist.location == normalize_path(os.getcwd())
```

Comment:
Will `dist.location` be always normalised? Or is it the case we should introduce an extra `or`?

Reply:
Seems to not work in linux. let me check it again...